test/eth.accounts.encrypt-decrypt.js

```diff
@@ -178,19 +194,24 @@ describe("eth", function () {
                             mac: '23d4497c779a6bc421f5cc54358309228389597f594448c5c900ad747f97401b' } };
 
                 var acc = ethAccounts.decrypt(encrypt, pw);
-                var ethWall = ethereumWallet.fromV3(encrypt, 'test');
+                var ethWall = await ethers.Wallet.fromEncryptedJson(JSON.stringify(encrypt), pw);
 
                 // compare addresses
-                assert.equal(acc.address, ethWall.getChecksumAddressString());
+                assert.equal(acc.address, ethWall.address);
                 assert.equal(web3.utils.toChecksumAddress('0x143f8913e0417997304fc179b531ff4cb9cab582'), acc.address);
-            });
+                assert.equal(web3.utils.toChecksumAddress('0x143f8913e0417997304fc179b531ff4cb9cab582'), ethWall.address);
+                assert.equal(ethers.utils.getAddress('0x143f8913e0417997304fc179b531ff4cb9cab582'), acc.address);
+                assert.equal(ethers.utils.getAddress('0x143f8913e0417997304fc179b531ff4cb9cab582'), ethWall.address);
 
+                // compare private keys
+                assert.equal(acc.privateKey, ethWall.privateKey);
+            });
         });
 
         staticTests.forEach(function (test, i) {
             it("decrypt staticTests and compare to private key", function() {
-                // increase the test timeout
-                this.timeout(6000);
+                // disable the test timeout
+                this.timeout(0);
```

Comment:
This should be removed not set to 0.

Reply:
Setting to `0` is the way to say "no timeout" in mocha: https://mochajs.org/#suite-level.

Some of the static tests have large `n` values: https://github.com/michaelsbradleyjr/web3.js/blob/d7adf702f815537376e7b8adb8fbdacfece17435/test/eth.accounts.encrypt-decrypt.js#L32.

In the case the Node.js being used falls back to `scryptsy`, which will be the case in CI for `<10.5.0` (since no `scrypt`) and also `>=10.5.0` because of maxmem errors with large `n` (until [PR #28799](https://github.com/nodejs/node/pull/28799) makes it into releases of 10.x/12.x), then those tests will be prone to timeouts (`scryptsy` is much, much slower for large `n`). So for these particular tests, it makes sense to disable the timeout.